docs/nv2/demo-script.md

```diff
@@ -217,7 +174,134 @@ To validate an image, `docker pull` with `docker notary --enabled` will attempt
+    -k wabbit-networks.key \
+    -c wabbit-networks.crt \
+    --plain-http \
+    --push \
+    --push-reference oci://${REPO}@$(oras discover \
+      --artifact-type application/x.example.sbom.v0 \
+      --output-json \
+      --plain-http \
+      $IMAGE | jq -r .references[0].digest) \
+    file:sbom-manifest.json
+  ```
+- Dynamically get the SBoM digest
+  ```bash
+  DIGEST=$(oras discover \
+      --artifact-type application/x.example.sbom.v0 \
+      --output-json \
+      --plain-http \
+      $IMAGE | jq -r .references[0].digest)
+- Discover referenced artifacts of the SBoM
+  ```bash
+  oras discover \
+    --plain-http \
+    ${REPO}@${DIGEST}
+  ```
+- Generates:
+  ```bash
+  Discovered 1 artifacts referencing localhost:5000/net-monitor@sha256:adfe3a3c50838fc2a19d5d7e73119dcadad7ad8e4e98f1e0fd100dd9d2278b71
+  Digest: sha256:adfe3a3c50838fc2a19d5d7e73119dcadad7ad8e4e98f1e0fd100dd9d2278b71
+
+  Artifact Type                    Digest
+  application/vnd.cncf.notary.v2   sha256:b7fc5fdb81f2ada359d0a709004360d1f08c9d2ac8a80630b152d1c6fb35460e
+  ```
+
+The above workflow demonstrates the **Notary v2, prototype-2** target experience.
+
+## Optional Steps
+
+Some optional steps:
+
+## Demo Reset
+
+If iterating through the demo, these are the steps required to reset to a clean state:
+
+- Remove docker alias:
+  ```bash
+  unalias docker
+  ```
+- Reset the local registry:
+  ```bash
+  docker rm -f $(docker ps -a -q)
+  docker run -d -p ${PORT}:5000 notaryv2/registry:nv2-prototype-2
+  ```
+- Remove the `net-monitor:v1` image:
+  ```bash
+  docker rmi -f ${REGISTRY}/net-monitor:v1
+  ```
+- Remove `wabbit-networks.crt` from `"verificationCerts"` in the `nv2.json` configuration file:
+  ```bash
+  code ~/.docker/nv2.json
+  ```
+- Remove previous signatures:
+  ```bash
+  rm -r ~/.docker/nv2/sha256/
+  ```
+
+### Simulating a Registry DNS Name
+
+Configure the additional steps to simulate a fully qualified dns name for wabbit-networks.
 
-[nv2-signature]:    ../signature/README.md
-[docker-generate]:  https://github.com/shizhMSFT/docker-generate
\ No newline at end of file
+- Setup names and variables with `registry.wabbit-networks.io`
+  ```bash
+  export PORT=80
+  export REGISTRY=registry.wabbit-networks.io
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Edit the `~/.docker/nv2.json` file to support local, insecure registries
+  ```json
+  {
+    "enabled": true,
+    "verificationCerts": [
+    ],
+    "insecureRegistries": [
+      "registry.wabbit-networks.io"
+    ]
+  }
+  ```
+- Add a `etc/hosts` entry to simulate pushing to registry.wabbit-networks.io
+  - If running on windows, _even if using wsl_, add the following entry to: `C:\Windows\System32\drivers\etc\hosts`
+    ```hosts
+    127.0.0.1 registry.wabbit-networks.io
+    ```
+- Continue with [Start a Local Registry Instance](#start-a-local-registry-instance)
+
+[docker-generate]:        https://github.com/shizhMSFT/docker-generate
```

Comment:
Should docker-generate refer to @shizhMSFT's repository or the local copy?

Reply:
nope, great catch. This got merged into: https://github.com/notaryproject/nv2/tree/prototype-2